psp/exp_configs/mone.py

```diff
@@ -11,13 +13,19 @@
 from psp.typings import Horizons
 
 _PREFIX = "/mnt/storage_b/data/ocf/solar_pv_nowcasting/clients/mone"
-PV_DATA_PATH = _PREFIX + "/pv.nc"
-PV_DATA_PATH_5MIN = _PREFIX + "/pv_5min.nc"
-NWP_DATA_PATH = (
-    "/mnt/storage_ssd_8tb/data/ocf/solar_pv_nowcasting"
-    "/nowcasting_dataset_pipeline/NWP/UK_Met_Office/UKV/zarr"
-    "/UKV_intermediate_version_7.zarr"
-)
+PV_DATA_PATH = _PREFIX + "/pv_v3.nc"
+PV_DATA_PATH_5MIN = _PREFIX + "/pv_v3_5min.nc"
+NWP_DATA_PATHS = [
+    (
+        "/mnt/storage_ssd_8tb/data/ocf/solar_pv_nowcasting"
```

Comment:
assuming these are just split up to look better in the code?

Reply:
Yeah so that the lines are not too long.